### DISCUSSION TITLE
Error after adding CommunityToolkit.Maui nuget to the project

### DISCUSSION THREAD

**Discussion by GabrieleMessina** [CommunityToolkit/Maui#380] [Q&A] [answered]
Hi, i've created a new MAUI (RC-1) project from Visual Studio template and after adding CommunityToolkit.Maui to my project i get this error:

``` shell
1>------ Build started: Project: TestNewMauiTemplateWithShell, Configuration: Debug Any CPU ------
1>You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
1>Skipping analyzers to speed up the build. You can execute 'Build' or 'Rebuild' command to run analyzers.
1>TestNewMauiTemplateWithShell -> \bin\Debug\net6.0-windows10.0.19041\win10-x64\TestNewMauiTemplateWithShell.dll
1>WINAPPSDKGENERATEPROJECTPRIFILE : error : PRI175: 0x80073b0f - Processing Resources failed with error: Voce duplicata.
1>WINAPPSDKGENERATEPROJECTPRIFILE : error : PRI222: 0x80073b0f - Unspecified error occurred.
1>Done building project "TestNewMauiTemplateWithShell.csproj" -- FAILED.
``` 

I have no other NuGet package in this project except for those needed by MAUI, what could be the problem?

**Comment by pictos** [comment, marked as answer]
@GabrieleMessina thanks for reporting this issue.
Could you attach a small repro?

**Reply by GabrieleMessina** [reply]
Sure, https://github.com/GabrieleMessina/CommunityToolkitMaui380

**Reply by pictos** [reply]
@GabrieleMessina 
I found this error
```
C:\Program Files\dotnet\packs\Microsoft.Maui.Resizetizer.Sdk\6.0.300-rc.1.5355\targets\Microsoft.Maui.Resizetizer.targets(571,9): error : ApplicationId 'com.companyname.communitytoolkitmaui380' was not a valid GUID. Windows apps use a GUID for an application ID instead of the reverse domain used by Android and/or iOS apps.
```

can you double check that?

**Reply by GabrieleMessina** [reply]
I don't have that error, I checked the .csproj (which is the one generated by the VS template) and actually there is this ApplicationId, I tried to change it to a GUID but nothing has changed.

**Reply by pictos** [reply]
Are you sure that you have all the tooling in the latest version? The issue is caused by an incorrect csproj tag.
https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj#L17

you shouldn't use the `ApplicationGUID` but use the `ApplicationId` instead. you can see the correct tag at the link above

**Reply by GabrieleMessina** [reply]
Apparently i have version 6.0.300-preview.22204.3 of dotnet sdk, i think 6.0.300-rc.1.5355 is required? I don't know why this happened as i downloaded the workload from the Visual Studio installer. How can i manually get the new version of the SDK? (visual studio is up to date)

**Reply by VladislavAntonyuk** [reply]
Your preview version is OK. 
Seems like your project has template from RC2 at least. ApplicationIdGuid for some reason doesn’t work in RC1 despite it was announced as merged. Also you have Tizen. Did you add it manually?

please check the .NET MAUI version you have. For now we support RC1.

**Reply by GabrieleMessina** [reply]
You are right but i don't know why, do you know how to downgrade to RC1?

**Reply by VladislavAntonyuk** [reply]
Uninstall maui workload (dotnet workload uninstall MAUI if I am not mistaken) and install it again using Visual Studio installer

**Reply by GabrieleMessina** [reply]
I fixed uninstalling the workloads and manually installing the RC1 following this instruction: https://github.com/dotnet/maui/blob/main/.github/DEVELOPMENT.md#rc-1-branch

Thank you for the help :)

**Comment by CliffAgius** [comment]
Closed as answered
